### ISSUE TITLE
3 out of the 4 examples don't work

### ISSUE THREAD

**Issue by foobarbecue** [tshort/openmodelica-javascript#4]
http://tshort.github.io/mdpad/mdpad.html?Modelica.Electrical.Analog.Examples.ChuaCircuit.md works, but I get blank pages with 404s for the other 3 examples.

**Comment by tshort** [comment]
These pages are really old, and I don't have an environment set up to reproduce them. I should take them down or put up a warning.

**Comment by foobarbecue** [comment]
Yeah, maybe a warning would be good. You never know when somebody might stop by and fix it for ya
